daemon/info.go

```diff
@@ -26,12 +26,16 @@ import (
 // SystemInfo returns information about the host server the daemon is running on.
 func (daemon *Daemon) SystemInfo() (*types.Info, error) {
 	kernelVersion := "<unknown>"
-	if kv, err := kernel.GetKernelVersion(); err == nil {
+	if kv, err := kernel.GetKernelVersion(); err != nil {
+		logrus.Errorf("Could not get kernel version: %v", err)
```

Comment:
unknown is a good way.  However I think it is not enough. If we get <unknown>, we should find a place to continue searching root cause or more information. Since the code returns err, maybe log this error can make administrator have more information in detail.